File: iotcloud-restapi-java/Core/src/com/qcloud/restapi/iotcore/request/GetMultiDevicesRequest.java
package com.qcloud.restapi.iotcore.request;

public class GetMultiDevicesRequest extends Request {

    /**
     * 构造GetMultiDevicesRequest对象
     *
     * @param productID 服务端应答返回的productID
     * @param taskID 服务端应答返回的taskID（可从GetCreateMultiDevTaskResponse对像中获取taskID）
     */
    public GetMultiDevicesRequest(String productID, String taskID) {
        put("Action", "GetMultiDevices");
        put("productID", productID);
        put("taskID", taskID);
        put("pageSize", "10");
        put("pageNum", "1");
    }

    /**
     * 构造GetMultiDevicesRequest对象
     *
     * @param taskID 服务端应答返回的taskID（可从GetCreateMultiDevTaskResponse对像中获取taskID）
     * @param pageSize
     * @param pageNum
     */
    public GetMultiDevicesRequest(String taskID, int pageSize, int pageNum) {
        put("Action", "GetMultiDevices");
        put("taskID", taskID);
        put("pageSize", String.valueOf(pageSize));
        put("pageNum", String.valueOf(pageNum));
    }
}

File: iotcloud-restapi-php/Core/src/IoTCloud/Request/GetMultiDevicesRequest.php
<?php

namespace TXIoTCloud\Request;

require_once "GetCreateMultiDevTaskRequest.php";

class GetMultiDevicesRequest extends GetCreateMultiDevTaskRequest
{

    /**
     * 分页页数
     */
    private $pageNum;

    /**
     * 分页大小，每页返回设备个数
     */
    private $pageSize;

    /**
     * GetCreateMultiDevTaskRequest constructor.
     * @param $timestamp   当前unix时间戳
     * @param $nonce       随机正整数，与timestamp联合起来，用于防止重放攻击
     * @param $productID   产品ID
     * @param $taskID      任务ID
     * @param $pageNum     分页页数
     * @param $pageSize    分页大小
     */
    public function __construct($timestamp, $nonce, $productID, $taskID, $pageNum, $pageSize)
    {
        parent::__construct($timestamp, $nonce, $productID, $taskID);
        $this->pageNum = $pageNum;
        $this->pageSize = $pageSize;
    }

    /**
     * 获取分页的大小
     */
    public function getPageSize()
    {
        return $this->pageSize;
    }

    /**
     * 设置分页的大小
     * @param $pageSize
     */
    public function setPageSize($pageSize)
    {
        $this->pageSize = $pageSize;
    }

    /**
     * 获取请求的页数
     */
    public function getPageNum()
    {
        return $this->pageNum;
    }

    /**
     * 设置请求的页数
     * @param $pageNum
     */
    public function setPageNum($pageNum)
    {
        $this->pageNum = $pageNum;
    }

}